### PR TITLE
Descriptions, README, Setup Guide

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/158343/lansweeper-descriptions

### CODE REVIEW

File: DESCRIPTION.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-My App
+Description
 ======
 
-todo: add a description that will appear alongside your app in Deskpro admin
+***Link devices to users to tickets:*** Linking devices to users in the app lets agents easily view information from Lansweeper about your user's device while interacting with them, making communication more efficient.
```
Comment (HappyPaul55) [inline DESCRIPTION.md:4]: Not sure this is correct, "Link devices to users to tickets". Is that meant to be something like "Link devices to users and/or tickets"

File: DESCRIPTION.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-My App
+Description
```
Comment (HappyPaul55) [inline DESCRIPTION.md:1]: The template says "My App" so should this say "Lanswepper" instead of "Description"

File: README.md [hunk 3]
```diff
@@ -1,44 +1,9 @@
...
 
-Pull Requests
-===
-Every app deployment requires that the version property in the manifest.json file be updated to reflect the new app version. This is so Deskpro can detect changes and add/upgrade apps accordingly. As such, we've made altering versions easy by having CI make the actual version change for you. Here's what we do:
+Link your devices registered in Lansweeper to your Deskpro users, powering better IT support and communication
 
-* We increment patch versions, i.e. 1.0.1, automatically. You don't need to do anything and this will happen
-* Minor versions, i.e. 1.1.0, are incremented if you add the minor-version GitHub label to your PR
-* Major versions, i.e. 2.0.0, are incremented if you add the major-version GitHub label to your PR
+![Lansweeper App - Deskpro](./docs/assets/screenshot.png)
```
Comment (HappyPaul55) [inline README.md:7]: File doesn't exist.

File: SETUP.md [hunk 4]
```diff
@@ -1,4 +1,42 @@
-My App Setup Instructions
+Lansweeper Setup Instructions
 ===
 
-TODO: Write setup instructions
+To install the Lansweeper App, you must first create an app with a "Client ID" and "Client Secret".
```
Comment (HappyPaul55) [inline SETUP.md:4]: This doesn't say where. I assume this is in Deskpro.

File: SETUP.md [hunk 5]
```diff
@@ -1,4 +1,42 @@
...
+
+In the opened window, select __"Cloud Application"__.
+
+[![](/docs/setup/setup-lansweeper-04.png)](/docs/setup/setup-lansweeper-04.png)
+
+Then, click on __"Continue"__.
+
+Here, we fill out the following fields:
+1. Provide the __"Application name"__ as "Deskpro App" (or any preferred name);
+2. Copy the __"Сallback URL"__ on the "Settings" tab and paste it into the __"Allowed callback URL"__
```
Comment (HappyPaul55) [inline SETUP.md:26]: > Copy the __"Сallback URL"__ on the "Settings" tab in Deskpro and paste it into the __"Allowed callback URL"__

File: SETUP.md [hunk 6]
```diff
@@ -1,4 +1,42 @@
...
+3. Choose __"Other"__ in the Integration dropdown
+4. In the __"Custom Integration Name"__ field, you can enter "Deskpro App" the same as in the "Application name" field.
+5. The __"Developed by"__ and __"Description"__ fields can be left blank. However, feel free to fill them in as you see fit.
+
+[![](/docs/setup/setup-lansweeper-05.png)](/docs/setup/setup-lansweeper-05.png)
+
+After filling in the fields, click __"Add Application"__. Then save the generated credentials and store them in a secure place.
+
+Ok, head back to Deskpro and enter your __Client ID__ and __Client secret__ into the app settings form.
```
Comment (HappyPaul55) [inline SETUP.md:35]: ... which you got from the generated credentials in Lansweeper.

**Review by HappyPaul55 — APPROVED** [review]

File: DESCRIPTION.md (hunk 1 above)
Comment (zpawn) [reply DESCRIPTION.md:4]: Everything is correct here. This application works in the user sidebar. Therefore, devices can only be linked to the user.

File: README.md (hunk 3 above)
Comment (zpawn) [reply README.md:7]: fixed

File: SETUP.md (hunk 4 above)
Comment (zpawn) [reply SETUP.md:4]: Added a sentence.

File: SETUP.md (hunk 6 above)
Comment (zpawn) [reply SETUP.md:35]: added

File: DESCRIPTION.md (hunk 2 above)
Comment (zpawn) [reply DESCRIPTION.md:1]: No needed

**Pull request merged**